addon/-private/ember-internals.js

```diff
@@ -1,22 +1,25 @@
-import __EMBER_METAL__ from '@ember/-internals/metal/index';
+import * as EMBER_METAL from '@ember/-internals/metal/index';
```

Comment:
I think we can use the named exports `descriptorForDecorator` and `isClassicDecorator`.

Reply:
updated